src/Control/FunFlow/ContentStore.hs

```diff
@@ -207,6 +223,14 @@ open :: Path Abs Dir -> IO ContentStore
 open storeRoot = do
   createDirIfMissing True storeRoot
   storeLockFd <- createFile (fromAbsFile $ lockPath storeRoot) ownerWriteMode
+  storeDb <- SQL.open (fromAbsFile $ dbPath storeRoot)
```

Comment:
I wonder whether we should include version information here from the start - it's the sort of thing which can be annoying to hack in later if we want to handle upgrades of the metadata DB. Then again, I suppose we haven't cared about updates to the actual store yet, so maybe not worth thinking about yet.

Reply:
I agree, we should version the db and the store itself. For the db in particular I wanted to look through available libraries on hackage first.